ui-lib/src/main/java/co/electriccoin/zcash/ui/screen/onboarding/view/OnboardingView.kt

```diff
@@ -63,20 +76,35 @@ private const val IS_NAVIGATION_IN_APP_BAR = false
  * @param onImportWallet Callback when the user decides to import an existing wallet.
  * @param onCreateWallet Callback when the user decides to create a new wallet.
  */
+@OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun Onboarding(
     onboardingState: OnboardingState,
+    isDebugMenuEnabled: Boolean,
     onImportWallet: () -> Unit,
-    onCreateWallet: () -> Unit
+    onCreateWallet: () -> Unit,
+    onFixtureWallet: () -> Unit,
 ) {
-    Column {
-        OnboardingTopAppBar(onboardingState)
-        OnboardingMainContent(onboardingState, onImportWallet = onImportWallet, onCreateWallet = onCreateWallet)
+    Scaffold(
```

Comment:
Ha, I wanted to do the same (wrap it to Scaffold instead of the Column) in my Compose Material3 PR, but I saw you already did it 🙂